georocket-server/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's the difference between this file and the logback.xml? Is this one used for unit tests?